src/models.py

```diff
@@ -0,0 +1,74 @@
+"""
+Definition of trades, orders, and executions.
+"""
+from __future__ import annotations
+from fractions import Fraction
+from dataclasses import dataclass
+
+
+@dataclass
+class Trade:
+    """
+    Class for executed orders.
+    """
+
+    data: OrderData
+    execution: OrderExecution
+
+    def adapt_execution_to_gas_price(self, gas_price, gas_price_adapted):
+        """
+        Given an executed order which was executed ata time with gas price `gas_price`, adapt the
```

Comment:
small typo in comment "...was executed ata time..."

Reply:
Fixed.